client/src/App.tsx

```diff
@@ -1,24 +1,34 @@
 import { useEffect, useState } from 'react'
 import SnippetCard from './components/SnippetCard'
-import { SnippetType } from './types'
+import { SnippetType, UserType } from './types'
 
 import TestForm from './components/TestForm'
+import { text } from 'stream/consumers'
 
 export default function App() {
   const [snippets, setSnippets] = useState<SnippetType[]>([])
   const [filter, setFilter] = useState<string>('')
+  // holds users added by the form - sk
+  const [users, setUsers] = useState<UserType[]>([])
+
 
   useEffect(() => {
     fetch('/api/snippets')
       .then(r => r.json())
       .then(d => setSnippets(d))
   }, [])
 
+  //Adds newly added user to stateful users object - sk
+  const handleAddNewUser = (name: string, email: string ) => {
```

Comment:
Could this take in UserType  instead of defining a new name, email?

Reply:
Certainly!